db/migrations/20190711091456_create_flop_storage_tables.sql

```diff
@@ -194,9 +227,14 @@ CREATE INDEX flop_kicks_header_id_index
     ON maker.flop_kicks (header_id);
 CREATE INDEX flop_kicks_address_index
     ON maker.flop_kicks (address_id);
+CREATE INDEX flop_kicks_kicks_index
+    ON maker.flop_kicks (kicks);
+
+COMMENT ON TABLE maker.flop_kicks
+    IS E'@name flopKicksStorage\n@omit';
 
 -- prevent naming conflict with maker.flop_kick in postgraphile
-COMMENT ON TABLE maker.flop_kicks IS E'@name flopKicksStorage';
+-- COMMENT ON TABLE maker.flop_kicks IS E'@name flopKicksStorage';
```

Comment:
same with this comment, maybe we can remove it since the renaming is being handled on line 234?